### DISCUSSION TITLE
亦搜移植规则，正文报错，希望有大佬能修一下

### DISCUSSION THREAD

**Discussion by Yuki-Zyj** [xiaohucode/yidaRule#198] [General]
[路漫漫.yds.txt](https://github.com/user-attachments/files/16242722/yds.txt)
![Screenshot_2024-07-16-11-14-59-652_com xinqiu yidaapp](https://github.com/user-attachments/assets/cf3a09ff-fc32-47c3-89c9-bc214e0952a0)

**Comment by Yuki-Zyj** [comment]
不行的话，还是直接嗅探吧
[路漫漫(已修复).yds.txt](https://github.com/user-attachments/files/16243189/yds.txt)

**Comment by xiaohucode** [comment]
css解析错误，不支持contains
